src/xrpld/app/tx/detail/LoanPay.cpp

```diff
@@ -278,7 +279,9 @@ LoanPay::doApply()
                    asset,
                    tenthBipsOfValue(debtTotalProxy.value(), coverRateMinimum),
                    loanScale) &&
-            !isDeepFrozen(view, brokerOwner, asset);
+            !isDeepFrozen(view, brokerOwner, asset) &&
+            requireAuth(view, asset, brokerOwner, AuthType::StrongAuth) ==
+            tesSUCCESS;
```

Comment:
As a `TER` cast to `bool`, `tesSUCCESS` evaluates to false, so this could be written as:

```suggestion
            !requireAuth(view, asset, brokerOwner, AuthType::StrongAuth);
```